datastore/datastore_test.go

```diff
@@ -3081,6 +3081,103 @@ func TestPutMultiTypes(t *testing.T) {
 	}
 }
 
+func TestGetWithReadTime(t *testing.T) {
+	type Ent struct {
```

Comment:
Le's not export the type, especially since it's test only. Also, would just one field be enough? The second doesn't really alter the meaning of the test and seems unnecssary.

Reply:
Ditto below.

Reply:
Done.